app/jobs/email_application_job.rb

```diff
@@ -1,16 +1,14 @@
 class EmailApplicationJob < ApplicationJob
-  def perform(snap_application_id:)
-    @snap_application = SnapApplication.find(snap_application_id)
-    ApplicationMailer.snap_application_notification(
-      file_name: snap_application.pdf.path,
-      recipient_email: snap_application.email,
-    ).deliver
-  ensure
-    snap_application.pdf.try(:close)
-    snap_application.pdf.try(:unlink)
-  end
+  def perform(export_id:)
+    export = Export.find(export_id)
 
-  private
+    export.execute do |snap_application|
+      ApplicationMailer.snap_application_notification(
+        file_name: snap_application.pdf.path,
+        recipient_email: snap_application.email,
+      ).deliver
 
-  attr_reader :snap_application
+      "Emailed to #{snap_application.email}"
```

Comment:
is this a logging statement? if so, do we want to explicitly pass it to `Rails.logger` ?